about.html

```diff
@@ -0,0 +1,120 @@
+                    <p class="mail-details email"><b>otijoseph305@gmail.com</b></p>
+                  </div>
+               </div>
+           </div>
+        </section>
+        <section class="our-logo">
+            <div class="main-container">
+                <div class="logo-style">
+                    <p class="mail-details"><b>Ifeanyi305 Virtual Learning in Schools: Pros and Cons.</b></p>
+                    <hr class="hr-line">
+                    <div class="offer">
+                        <p class="mail-details">Over the course of this workshop, we will explore the online learning experience for students.</p>
+                    </div>
+                    <div class="my-logo logo-center">
+                        <a href="#"><img class="logo" src="images/Ifeanyi_Logo_1B-01-removebg-preview.png" alt=""></a>
+                    </div>
+                </div>
+                <div class="past-students">
+                    <p class="mail-details"><b>See our past students</b></p>
+                    <hr class="hr-line">
+                    <p class="mail-details">View our successful students with the highest level in 2020 and 2021, currently running.</p>
+                    <div class="student-img">
+                        <img src="images/benkler-small-square-250x250.png" alt="">
+                        <img src="images/imagesstudent.jpg" alt="">
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="connect-sections">
+            <div class="main-container">
+                <span class="main-program2">Main program</span>
+                <hr class="hr-line">
+                <div class="connect-img">
+                    <img class="the-img" src="images/mozilla.png" alt="">
+                    <img class="the-img" src="images/googlekorea.png" alt="">
+                    <img class="the-img" src="images/naver.png" alt="">
+                    <img class="the-img" src="images/kakao-2.png" alt="">
+                    <img class="the-img" src="images/airbnb-2.png" alt="">
+                </div>
+            </div>
+        </section>
+        <section class="footer" id="footer">
+            <div class="main-container">
+                <div class="footer-con">
+                    <div class="my-logo">
+                        <a href="#"><img class="logo" src="images/Ifeanyi_Logo_1B-01-removebg-preview.png" alt=""></a>
+                    </div>
+                    <div>
+                        <p class="down" id="changes">2022 Creative Commons Korea. Some Rights Reserved.</p>
+                        <p id="changes">All content on the website, unless otherwise specified, is available under a Creative Commons Attribution 2.0 Korea License. Please see http://creativecommons.org/licences/by/2.0/kr</p>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
- [ ] Kindly align your footer section to match the given design.

Your design
![image](https://user-images.githubusercontent.com/81932478/196735002-58812a43-9211-4eee-b870-02513c29aed6.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/196735115-4de06d34-42aa-4780-8eb7-d73427d1985b.png)